src/util.h

```diff
@@ -43,7 +47,17 @@ static SECP256K1_INLINE void secp256k1_callback_call(const secp256k1_callback *
 #define EXPECT(x,c) (x)
 #endif
 
-#ifdef DETERMINISTIC
+/* CHECK() is like assert(). We use it only in the tests. */
+#if defined(COVERAGE)
+/* Don't abort in coverage mode.
+   This avoids branches which are not expected to be taken.
+   We still use cond as usual to avoid unused variable warnings. */
```

Comment:
s/unused variable/unused return values/

Reply:
Perhaps also note that this works because an if statement with an empty consequent is not treated as a branch.